r/R/enums.R

```diff
@@ -135,7 +135,9 @@ FileType <- enum("FileType",
 #' @export
 #' @rdname enums
 ParquetVersionType <- enum("ParquetVersionType",
-  PARQUET_1_0 = 0L, PARQUET_2_4 = 2L, PARQUET_2_6 = 3L
+  PARQUET_1_0 = 0L, PARQUET_2_4 = 1L, PARQUET_2_6 = 2L,
+  PARQUET_2_7 = 3L, PARQUET_2_8 = 4L, PARQUET_2_9 = 5L, PARQUET_2_10 = 6L,
+  PARQUET_2_11 = 7L, PARQUET_2_12 = 8L
```

Comment:
Apparently this change ensures it matches up with the C++; not sure about this myself.